client/src/components/RegisterUser.jsx

```diff
@@ -48,19 +48,20 @@ class RegisterUser extends React.Component {
 
     return (
       <div>
-        <h2>Register a new User</h2>
+        <h1>Bite Share</h1>
```

Comment:
100% future thinking: Make the Bite Share heading into a component, so we can keep things DRY. 100% unimportant now